kubernetes/onpremise/kubeadm/scripts/install_kubernetes.sh

```diff
@@ -0,0 +1,177 @@
+# install Kubectl
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+curl -LO "https://dl.k8s.io/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl.sha256"
+echo "$(<kubectl.sha256) kubectl" | sha256sum --check
```

Comment:
Make it POSIX compliant:

```suggestion
echo "$(cat kubectl.sha256) kubectl" | sha256sum --check
```